tests/oeedger8r/edl/aliasing.edl

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do you want to have separate edl files or just a single edl file for locking down bug fixes?

Reply:
This one should be separate because we have to build with specific flags to repro the bug.

Reply:
Ah ok.